docs/installation.rst

```diff
@@ -560,6 +561,156 @@ As mentioned before, the script engine can be started from within ``$SCRIPT_ENGI
+
+Create an email domain for the mail relay
+`````````````````````````````````````````
+In order to work properly, the mail relay needs to receive all relevant emails.
+The used email addresses are created dynamically based on the ID of the workflow that should be started.
+The domain used for the email addresses can be chosen by you and has to be configured in the system. 
+We advise that you create a specific subdomain which is different from your normal email domain and redirect any incoming emails to the mail relay.
+
+For example if you use the email domain ``mail.yourcompany.com``, a typical email address for the workflow system would look like this: ::
+
+    process-5702854fd1dfff250dc57994@mail.yourcompany.com
+
+How you route the emails to the mail relay is up to you and depends on your infrastructure.
+If the server which runs the mail relay is publicly available, you can simply setup a respective MX record.
+Note, the mail relay and the Workflow Accelerator web application don't have to run on the same server.
+If your email server (e.g. Microsoft Exchange) allows to reroute traffic for a specific subdomain to another server, you can also setup a respective rule there.
+Please, make sure the email is not modified and for instance the sender address is not changed due to some forwarding mechanism.
+
+The next subsection explains how you setup the mail relay and configure it to use your email domain.
+
+Install and configure the mail relay
+````````````````````````````````````
+The mail relay comes as a JAR file which requires Java 8 for the execution.
+Please, make sure Java 8 is installed before continuing with the setup.
+
+The mail relay can be installed on the same server as the Workflow Accelerator web application or on a different server.
+If you choose two different servers, please ensure the mail relay is able to reach the server which runs the web application.
+
+#. Create a new local directory for the mail relay, e.g. ``C:\Program Files\Mail Relay`` or ``/var/lib/mail-relay``.
+
+    * We will refer to this directory as ``$MAIL_RELAY_HOME``.
+
+#. Copy the content of the directory ``$WORKFLOW_HOME/mail-relay/`` to your newly created directory.
+
+    * You should find the file ``mail-relay.jar`` directly within your directory: ``$MAIL_RELAY_HOME/mail-relay.jar``.
+
+#. Within the ``$MAIL_RELAY_HOME`` directory create a new directory ``logs``.
+
+#. Open ``$MAIL_RELAY_HOME/logback.xml`` and edit ``value`` in the following line: ::
+
+    <property name="LOG_DIR" value="."/>
+
+#. Set ``value`` to the absolute path of ``$MAIL_RELAY_HOME/logs``.
+
+    * For example ``C:\\Program Files\\Mail Relay\\logs`` or ``/var/lib/mail-relay/logs``
+    * On Windows you have to use ``\\`` as a separator for the path.
+
+#. Open ``$MAIL_RELAY_HOME/mail-relay.properties`` and edit the values for the domain, port and Workflow Accelerator URL.
+
+    * Ensure that the port for the mail relay is not used by any other application.
+    
```

Comment:
The numbered list is good: that was early feedback from Ngoc - that we should use numbered lists for instructions more.